zetaclient/context/app.go

```diff
@@ -127,6 +140,22 @@ func (a *AppContext) GetCrossChainFlags() observertypes.CrosschainFlags {
 	return a.crosschainFlags
 }
 
+// SetRelayerKeyPasswords sets the relayer key passwords for given networks
+func (a *AppContext) SetRelayerKeyPasswords(relayerKeyPasswords map[chains.Network]string) {
```

Comment:
This should be moved to app.Update()

Reply:
We'll need it to remember what passwords were entered when `zetaclientd` starts. The passwords are collected from user input and are static values that won't change/update.

Reply:
>We'll need it to remember what passwords were entered by user when zetaclientd starts

Then this should be done in AppContext constructor. Let's avoid having many entry points in AppContext's state.

Reply:
Also, can you elaborate on why validators need to enter the password manually? 

Can we parse it from `.passwords` file or take it from zetaclient `yml` config?

Reply:
I think it's for security purpose. The keys and password should not be in the form of plaintext in any files. @swift1337

Reply:
password map is moved to chain registry: https://github.com/zeta-chain/node/pull/2673/commits/6a4399ce1b26b4c772398a5c69895a63f2c47632

Reply:
app context constructor now takes the passwords as an argument: https://github.com/zeta-chain/node/pull/2673/commits/6a4399ce1b26b4c772398a5c69895a63f2c47632